src/renderer/extensions/vueNodes/components/NodeWidgets.vue

```diff
@@ -57,7 +57,14 @@
           :model-value="widget.value"
           :node-id="nodeData?.id != null ? String(nodeData.id) : ''"
           :node-type="nodeType"
-          class="flex-1 col-span-2"
+          :class="
+            cn(
+              'flex-1 col-span-2',
+              widget.borderColor && 'border rounded-lg',
+              widget.borderColor === 'promoted' && 'border-fuchsia-600/80',
+              widget.borderColor === 'advanced' && 'border-azure-400/80'
```

Comment:
These should probably be theme colors.

Also, could the whole composite set of classes be joined when the widget's status is polled and just put unconditionally?